cms/page_rendering.py

```diff
@@ -18,7 +19,10 @@ def render_page(request, page, current_language, slug):
     """
     template_name = get_template_from_request(request, page, no_current_page=True)
     # fill the context
-    context = RequestContext(request)
+    if DJANGO_1_7:
+        context = RequestContext(request)
```

Comment:
Rule of thumb I learnt a while ago: In django CMS, when in doubt, always use ``RequestContext`` :)
django CMS relies a lot on request being in the context